src/ceph-daemon

```diff
@@ -1189,6 +1188,8 @@ def command_ls():
             elif is_fsid(i):
                 fsid = i
                 for j in os.listdir(os.path.join(args.data_dir, i)):
+                    if '.' not in j:
```

Comment:
Seems dangerous to me. Can we make similar to
```suggestion
                    if not j.startswith('./'):
```
?

Reply:
this is really there to skip the 'crash' directory.  but probably we should be tolerant of 'a.b.c' too. will make this more robust